README.md

```diff
@@ -64,12 +64,11 @@ Clone the repository:
 git clone https://github.com/akermu/emacs-libvterm.git
 ```
 
-In case you want to use the version of libvterm already installed on your
-system, change `cmake ..` with `cmake -DUSE_SYSTEM_LIBVTERM=yes ..` in the
-following instructions. If `-DUSE_SYSTEM_LIBVTERM` is not explicitly set to
-`yes` (or if it is set to `no`), emacs-libvterm will download the latest version
-available of libvterm (from [here](https://github.com/neovim/libvterm)), compile
-it, and use it.
+By default, system libvterm is automatically detected. If it is not installed,
+emacs-libvterm will download the latest version available of libvterm (from
+[here](https://github.com/neovim/libvterm)), compile it, and use it. If you want
+to use the vendored version, set `ENABLE_LIBVTERM_AUTO_DETECTION` to `Off`. The
+option will be passed into `cmake` through `vterm-module-cmake-args`.
 
```

Comment:
I realized that the current setup is more complicated than it should be. 
I think what `vterm` should do is:
- Check if it can find libvterm
- If it can, use it _unless_ `USE_SYSTEM_LIBVTERM` is set to no. In this case, download libvterm
- If it cannot find system-libvterm, download it.

Hence, I suggest we remove the option for `ENABLE_LIBVTERM_AUTO_DETECTION` and change the way `USE_SYSTEM_LIBVTERM` is used.


```suggestion
By default, vterm will try to find if libvterm is installed. If it is 
not found, vterm will download the latest version available of libvterm (from 
[upstream](https://github.com/neovim/libvterm)), compile it, and use it. If you always
want to use this version as opposed to the one on your system, set `USE_SYSTEM_LIBVTERM` 
to `no`. To do this, change `cmake ..` with `cmake -DUSE_SYSTEM_LIBVTERM=no ..` in the
following instructions.
￼```